staging/src/k8s.io/apiserver/pkg/server/options/etcd.go

```diff
@@ -369,3 +359,56 @@ func WriteWatchCacheSizes(watchCacheSizes map[schema.GroupResource]int) ([]strin
 	}
 	return cacheSizes, nil
 }
+
+var _ serverstorage.StorageFactory = &StorageConfigFactory{}
+
+type StorageConfigFactory struct {
```

Comment:
a comment and/or name change to distinguish between this type and `DefaultStorageFactory` would be helpful to understand when one should be used over the other. Should this be `SimpleStorageFactory`?